src/scripts/main.js

```diff
@@ -1,3 +1,244 @@
 'use strict';
 
-// write your code here
+const startBtn = document.querySelector('.button');
+const messageLose = document.querySelector('.message-lose');
+const messageWin = document.querySelector('.message-win');
+const messageStart = document.querySelector('.message-start');
+const score = document.querySelector('.game-score');
+const gameScore = document.querySelector('.game-score');
+const rows = document.querySelectorAll('.field-row');
+
+let arrOfNumbers = [
+  ['', '', '', ''],
+  ['', '', '', ''],
+  ['', '', '', ''],
+  ['', '', '', ''],
+];
+let moves = 0;
+let maxNum = 0;
+
+score.innerText = maxNum;
```

Comment:
The initial score is set to maxNum which is 0. According to task requirements, the score should increase with each move, based on the sum of all merged cells.